code/modules/antagonists/morph/morph_stomach.dm

```diff
@@ -72,6 +72,9 @@
 /datum/morph_stomach/ui_act(action, params)
 	if(..())
 		return
+	if(morph.movement_type & VENTCRAWLING)
+		to_chat(morph, "<span class='danger'>You cannot do that while ventcrawling.</span></span>")
```

Comment:
```suggestion
		to_chat(morph, "<span class='danger'>It's too cramped, you can't digest this here!.</span></span>")
```

Reply:
It seems better to have an IC reason as to why they can't digest someone into a pipe

Reply:
replace digest with do